Assets/HoloToolkit/Utilities/Scripts/Inspectors/MRTKEditor.cs

```diff
@@ -1015,6 +1015,26 @@ protected Quaternion RotationHandle(Vector3 position, Quaternion rotation, float
             return rotation;
         }
         #endregion
+
+        #region editor prefs
+        
+        private static void SetEditorPref(string key, bool value)
+        {
+            EditorPrefs.SetBool(Application.productName + key, value);
```

Comment:
Why aren't you using the `EditorPrefsUtility`? 

This just duplicates existing code.

Reply:
It's compiled in the Editor project - it's not accessible outside of Editor folders.

Reply:
I really don't like the way we've engineered these editor windows.  This really breaks commonly used indusrty standard design paradigms.

Reply:
I understand. Again I'll invite you to open an issue where we can discuss the subject in depth & come up with alternatives. I'm happy to entertain any suggestions.

Reply:
Well by biggest concern here, specifically, is duplicated code.

Reply:
Right. Since EditorPrefsUtility lives in the Editor folder and isn't accessible to the class we're working with, there doesn't seem to be a way to avoid it, at least not without some major restructuring.

If you want to discuss that restructuring, let's try opening an issue where we can work out an alternative. I'm sure there's some better way that we just haven't found yet.

But if that doesn't interest you then we'll just have to live with 12 or so lines of duplicated code.

Reply:
I think it's fine for now.

Reply:
Sounds good.